backend-api/tests/api-tests/clientCredentialsGrantFlow.test.ts

```diff
@@ -7,325 +7,328 @@ import { randomUUID, UUID } from "crypto";
 import { PRIVATE_API_INSTANCE, PROXY_API_INSTANCE } from "./utils/apiInstance";
 import { AxiosInstance } from "axios";
 
-const APIS_TO_TEST: [
-  apiInstanceName: string,
-  apiInstace: AxiosInstance,
-  authorizationHeader: string,
-][] =
-  process.env.IS_LOCAL_TEST === "true"
-    ? [["Proxy API Instance", PROXY_API_INSTANCE, "x-custom-auth"]]
-    : [
-        ["Proxy API Instance", PROXY_API_INSTANCE, "x-custom-auth"],
-        ["Private API Instance", PRIVATE_API_INSTANCE, "Authorization"],
-      ];
-
-describe.each(APIS_TO_TEST)(
-  "Test %s",
-  (
-    apiInstanceName: string,
-    apiInstance: AxiosInstance,
-    authorizationHeader: string,
-  ) => {
-    let clientIdAndSecret: string;
+const getApisToTest = (): Array<{
+  apiName: string;
+  api: AxiosInstance;
```

Comment:
I think this may want to be called axiosInstance as it's not _really_ an api

Reply:
Good point...

Reply:
Done!

Reply:
nit(take it or leave it): stylistically in the team we adopt the shorthand syntax:

`{ key: value }[]`